graph.sh

```diff
@@ -11,8 +11,14 @@ if [ ! -d "output/graphs/$VERSION" ]; then
     exit 1
 fi
 
-echo "Creating images of graph data..."
 cd output/graphs/$VERSION
+
+if [ -z "$(ls -A | grep .gv)" ]; then
+    echo "No data to plot"
+    exit 1
```

Comment:
Is this an error condition or a normal condition? If an normal condition it should be `exit 0` otherwise it is correctly `exit 1`.

Reply:
You're right - it should be a normal exit, just without doing any processing